Assets/Scripts/Notebook/Button.cs

```diff
@@ -71,6 +71,9 @@ private void OnEnable()
             _teleportController = TeleportController.Instance;
             if (_teleportController == null) throw new System.Exception("Must have a teleport controller in the scene");
```

Comment:
Also here, what if you couldn't teleport in a scene? i also don't think a teleportController should be necessary

Reply:
I can null check for if an envi doesnt have a teleport controller but this is just so that in scenes where you can teleport, a laser doesnt appear when you try to press a button and accidentally move you